script.js

```diff
@@ -0,0 +1,116 @@
+const inputJs = document.querySelector(".input"),
+    toDoNewJs = inputJs.querySelector("#todo-new"),
+    listWaitingJs = document.querySelector(".waiting"),
+    listCompleteJs = document.querySelector(".complete");
+
+const TODOS = "toDos"; //key for saving todos in local storage
+
+let toDos = [];
```

Comment:
변수이름을 선언할 때 복수형보다는 todoList와 같은 형태로 선언해주시면 좋습니다~ [자바스크립트 네이밍 컨벤션 참고](https://ui.toast.com/fe-guide/ko_CODING-CONVENSION/#%EB%AA%85%EB%AA%85-%EA%B7%9C%EC%B9%99)
```suggestion
let todoList= [];
```

Reply:
네! 수정 했습니다 :)